deploy.sh

```diff
@@ -55,6 +55,9 @@ $THIS_DIR/scripts/update_project_version.py public_pom.xml $project_version > ge
 
 mvn deploy ${MVN_OPTIONS[@]} -Dossrh-deploy
```

Comment:
We shouldn't do `mvn deploy` before testing this on travis.

Reply:
good idea, I've updated this to `mvn package` for now.